edx_rest_framework_extensions/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '1.8.0'  # pragma: no cover
+__version__ = '1.9.0'  # pragma: no cover
```

Comment:
@nasthagiri: FYI: This is the pattern I follow for versioning: https://semver.org/
Only backward-compatible bug fixes should update the final digit.  Any new functionality (backward-compatible) should bump the middle number as I have done here.